pages/common/ogr2ogr.md

```diff
@@ -0,0 +1,24 @@
+# ogr2ogr
+
+> Converts simple features data between file formats.
+> More information: <https://gdal.org/programs/ogr2ogr.html#ogr2ogr>.
+
+- Convert a Shapefile into a GeoPackage:
+
+`ogr2ogr -f GPKG output.gpkg input.shp`
+
+- Reproject a GeoPackage:
+
+`ogr2ogr -s_srs EPSG:4326 -t_srs EPSG:3857 -f GPKG output.gpkg input.gpkg`
+
+- Convert a CSV file into a GeoPackage and assign projection:
+
+`ogr2ogr -f GPKG output.gpkg input.csv -oo X_POSSIBLE_NAMES=lon -oo Y_POSSIBLE_NAMES=lat -a_srs 'EPSG:4326'`
+
+- Load a GeoPackage into a PostGIS database:
+
+`ogr2ogr -f "PostgreSQL" PG:dbname="my_database" input.gpkg`
+
+- Filter file by bounding box (<xmin> <ymin> <xmax> <ymax>):
```

Comment:
```suggestion
- Clip layers of a GeoPackage file to the given bounding box:
```
"Filter" is not very accurate, what this command does is clipping. We can also move the names of the values in the command itself to make it both easier to read and to understand.